### PR TITLE
do not use matrix for node versions in tests action

### PR DESCRIPTION
**Pull request by interstella-5555**
do not run tests on different node versions (unnecessary)

### CODE REVIEW

**Review by peterjan — DISMISSED** [review]

File: .github/workflows/ci_release.yml [hunk 1]
```diff
@@ -26,16 +26,11 @@ jobs:
...
-      fail-fast: false
-      matrix:
-        node-version: [14.x, 16.x]
-
     steps:
       - name: "Run Yarn Basic Checks"
         uses: SkynetLabs/.github/.github/actions/yarn-basic-checks@master
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 18.x
```
Comment (meeh0w) [inline .github/workflows/ci_release.yml]: Looks like there is some issue with Node 18 (checks are failing). We should do `16` to be consistent with our `Dockerfile`

**Review by meeh0w — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_release.yml (hunk 1 above)
Comment (interstella-5555) [reply .github/workflows/ci_release.yml]: updated to 16.x in https://github.com/SkynetLabs/webportal-accounts-dashboard/pull/56/commits/9ecdbf6baab85249356f06de5ee17f8c484168fc

**Review by peterjan — APPROVED** [review]

**Review by meeh0w — APPROVED** [review]

**Comment by meeh0w** [comment]
I re-ran the checks as they succeeded, but didn't report their status for whatever reason.

**Comment by meeh0w** [comment]
Ohh..it's waiting for the old matrix checks to complete, but they won't run from this point on. Something got out of hand here. We'll probably need the repo admin to merge it manually (cc @MSevey).

**Pull request merged**